modules/python/nitf/source/generated/nitropy_wrap.cxx

```diff
@@ -34645,7 +34645,7 @@ SWIG_init(void) {
   
   SWIG_InstallConstants(d,swig_const_table);
   
-  SWIG_Python_SetConstant(d, "NRT_FILE",SWIG_FromCharPtr("/corp/SST/jmeans/workspace/nitro/modules/c/nrt/include/nrt/Defines.h"));
+  SWIG_Python_SetConstant(d, "NRT_FILE",SWIG_FromCharPtr("/data1/u/jmeans/nitro/modules/c/nrt/include/nrt/Defines.h"));
```

Comment:
Perhaps you've already thought of this, but it seems like the 'configure' step of the cmake build could be used generate a header with a define for the developer-specific path

```c++
// some_header.h
#define NITRO_MODULES_SOURCE "/data/u/jmeans" // derived from config

// this file
#include "some_header.h"
SWIG_Python_SetConstant(d, "NRT_FILE",SWIG_FromCharPtr(NITRO_MODULES_SOURCE "/nitro/modules/c/nrt/include/nrt/Defines.h"))
```

Reply:
I think we actually just don't want these values at all. See update.